pkgs/applications/video/shaka-packager/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv
+, fetchurl
+, lib
+, runCommand
+, shaka-packager
+, system ? builtins.currentSystem
+}:
+let
+  binary-download-details = {
+    x86_64-linux = {
+      filename = "packager-linux-x64";
+      sha256 = "13f8n83sphk6frwcbkk8xd7yalfsqaclcw3jl6jzkg1dy7l1g0rj";
+    };
```

```suggestion
    "x86_64-linux" = {
      filename = "packager-linux-x64";
      hash = "sha256-MoMX6PEtvPmloXJwRpnC2lHlT+tozsV4dmbCqweyyI0=";
    };
```

Run `nix hash to-sri sha256:13f8n83sphk6frwcbkk8xd7yalfsqaclcw3jl6jzkg1dy7l1g0rj` to modify to SRI hash.